addon/components/document-details.hbs

```diff
@@ -186,7 +186,7 @@
           <div class="uk-accordion-content">
             <ul class="uk-list uk-list-divider">
               {{#each (filter-by "type" "original" @document.files) as |file|}}
-                <li data-test-file class="uk-flex">
+                <li data-test-file class="uk-flex" title={{file.name}}>
```

Comment:
How does the `title=` work on a li element? I've never seen this before.

Reply:
Also didn't find anything on the MDN `li` doc.

Reply:
The title attribute is a global attribute. You can slap it on anything.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title

Reply:
:+1: